webapp/packages/core-browser/src/selectFiles.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+export function selectFiles(callback: (files: FileList | null) => any): void {
```

Comment:
Probably we need to attach the input to the body and set its style to be hidden. 
If it's not attached to the DOM, the click event might not work in some browsers.

Reply:
fixed